docs/source/howto/_plot_x_y_databroker.ipynb

```diff
@@ -563,22 +590,123 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
      },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "xyplot(dataset, \"m1\", \"noisy\", title=\"Example XY plot\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "d2052f64",
+   "metadata": {},
+   "source": [
+    "Alternatively, the dataset can be referenced directly from the databroker catalog object `cat`.  (Here, `cat[-1]` is the `run`.)\n",
```

Comment:
Earlier (cell #2) run was defined as:
run=cat.v2[86] 
This might be confusing. Why did you use v2 earlier?

Reply:
Rather than explain, removed the `.v2` part.